namespaces/mson-namespace.md

```diff
@@ -2,20 +2,37 @@
 
 This document extends [Refract][] Specification with new element types necessary to build [MSON][] DOM.
 
-# Content
-
-This namespace defines following elements:
-
-1. [MSON Element](#mson-element-element)
-1. [Boolean Type](#boolean-type-boolean-element)
-1. [String Type](#string-type-string-element)
-1. [Number Type](#number-type-number-element)
-1. [Array Type](#array-type-array-element)
-1. [Object Type](#object-type-object-element)
-1. [Property Type](#property-type-property-element)
-1. [Enum Type](#enum-type-mson-element)
-
-# About this Document
+## Content
+
+<!-- TOC depth:3 withLinks:1 updateOnSave:0 -->
```

Comment:
@kylef yes

https://atom.io/packages/markdown-toc

there is a sublime text version too, very handy

cc @smizell